usaspending_api/settings.py

```diff
@@ -169,36 +169,34 @@ def _configure_database_connection(environment_variable):
     Configure a Django database connection... configuration.  environment_variable is the name of
     the operating system environment variable that contains the database connection string or DSN
     """
-    default_options = {"options": "-c statement_timeout={0}".format(DEFAULT_DB_TIMEOUT_IN_SECONDS * 1000)}
+    default_options = {"options": "-c statement_timeout={0}".format(int(DEFAULT_DB_TIMEOUT_IN_SECONDS) * 1000)}
```

Comment:
@sethstoudenmier I think the fix should instead be on the module variable as its set, and casting that to an int when read from the environment. 

```
DEFAULT_DB_TIMEOUT_IN_SECONDS = os.environ.get("DEFAULT_DB_TIMEOUT_IN_SECONDS") or 0
```

This solution is not tested but maybe:
```
DEFAULT_DB_TIMEOUT_IN_SECONDS = int(os.getenv("DEFAULT_DB_TIMEOUT_IN_SECONDS", "0"))
```
or
```
DEFAULT_DB_TIMEOUT_IN_SECONDS = int(os.environ.get("DEFAULT_DB_TIMEOUT_IN_SECONDS", 0))
```

Reply:
This seems like the code you're looking for:
```python
DEFAULT_DB_TIMEOUT_IN_SECONDS = int(os.environ.get("DEFAULT_DB_TIMEOUT_IN_SECONDS")) or  0
```

Reply:
@tony-sappe what does `int()` do on a `None` object?

Reply:
`TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'`

Reply:
Tested... looks like it throws a `TypeError`

So the risk is an error if that env var is not set. It was safe the way Seth had it, because of the `None or 0` ... but when casting at the spot, it needs a default value to be provided on the `get` or `getenv`.

Reply:
yeah, my bad.
```python
DEFAULT_DB_TIMEOUT_IN_SECONDS = int(os.environ.get("DEFAULT_DB_TIMEOUT_IN_SECONDS", 0))
```

This will still throw an exception for strings which cannot be converted to an int. And I don't know what will happen with negative values.